pages/windows/attrib.md

```diff
@@ -0,0 +1,27 @@
+# attrib
+
+> Displays or changes file and directories attributes.
+
+- Display the permission of the files in current directory:
```

Comment:
You mention attributes in the description of the command, but you use the word permission here? This sounds a bit confusing.

Reply:
I think "attributes" sound better. Because we have system, hidden and all that.